docs/06-concepts/18-testing/04-best-practises.md

```diff
@@ -0,0 +1,103 @@
+# Best practises
+
+## Imports
+
+### Don't
+
+```dart
+import 'serverpod_test_tools.dart';
+// Don't import `serverpod_test` directly.
+import 'package:serverpod_test/serverpod_test.dart'; ❌  
+```
+
+### Do
+
+```dart
+// Only import the generated test tools file.
+// It re-exports all helpers and types that are needed.
+import 'serverpod_test_tools.dart'; ✅ 
+```
+
+### Database clean up
+
+Unless configured otherwise, by default `withServerpod` does all database operations inside a transaction that is rolled back after each `test`.
```

Comment:
Suggestion: Mention the configuration option that changes this behavior, if someone has not read that section of the documentation it can be good to point them there.